mteb/tasks/Classification/zul/IsiZuluNewsClassification.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 2800
+
+
+class IsiZuluNewsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="IsiZuluNewsClassification",
+        description="isiZulu News Classification Dataset",
+        reference="https://huggingface.co/datasets/dsfsi/za-isizulu-siswati-news",
+        dataset={
+            "path": "isaacchung/isizulu-news",
+            "revision": "55caf0e52693a1ea63b15a4980a73fc137fb862b",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["zul-Latn"],
```

Comment:
What about Siswati? (the code seems to be ssw)

Reply:
I only copied over the Zulu one for now. Will do the Siswati one separately. When loading the original dataset I encountered an error. :/

Reply:
Ahh right. Should they be in the same dataset?